alert-policies/apache-traffic-server/SSLExpiredCertificate.yml

```diff
@@ -0,0 +1,30 @@
+# Name of the alert
+name: SSL Expired Certificates
+
+# Description and details
+description: |+
+  This alert is triggered if the SSL expired certificates exceeds 1 for 5 minutes.
```

Comment:
```suggestion
  This alert is triggered when the SSL certificate is within 15 days of expiration.
```

Reply:
Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.